Reference/conformance/conformance.pb.swift

```diff
@@ -230,11 +230,28 @@ struct Conformance_ConformanceResponse: SwiftProtobuf.Message {
   var unknownFields = SwiftProtobuf.UnknownStorage()
 
   enum OneOf_Result: Equatable {
```

Comment:
So for this one, the property is _result_, we stick the _OneOf\__ on the front for the enum's name.